apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktService.kt

```diff
@@ -102,13 +108,48 @@ class DoedshendelseKontrollpunktService(
         beroert: PersonRolle,
     ): Triple<Sak?, PersonDTO, PersonDTO> {
         val sakType = hendelse.sakTypeForEpsEllerBarn()
-        val sak = sakService.finnSak(hendelse.beroertFnr, sakType)
+        val sak = hentSakForDoedshendelse(hendelse.beroertFnr, sakType)
+
         val avdoed = pdlTjenesterKlient.hentPdlModellForSaktype(hendelse.avdoedFnr, PersonRolle.AVDOED, sakType)
-        val gjenlevende = pdlTjenesterKlient.hentPdlModellForSaktype(hendelse.beroertFnr, beroert, hendelse.sakTypeForEpsEllerBarn())
+        val gjenlevende =
+            pdlTjenesterKlient.hentPdlModellForSaktype(hendelse.beroertFnr, beroert, hendelse.sakTypeForEpsEllerBarn())
 
         return Triple(sak, avdoed, gjenlevende)
     }
 
+    /**
+     * Midlertidig løsning inntil vi får nøstet opp i tilfellene hvor det finnes flere saker på én person
+     * Gjelder i hovedsak personer som har fått endret identifikator
+     **/
+    private fun hentSakForDoedshendelse(
+        beroertFnr: String,
+        sakType: SakType,
+    ): Sak? =
+        try {
+            sakService.finnSak(beroertFnr, sakType)
+        } catch (e: InternfeilException) {
+            val saker = sakService.finnSaker(beroertFnr)
+
+            if (saker.size > 1) {
```

Comment:
Si det her skjer i produksjon, vil vi ikke potensielt nå da sende ut et brev selvom bruker allerede har sendt en søknad?

Reply:
Hmm, men er det vanlig at det sendes ut dødshendelser flere ganger på samme person? 

Tenkte kanskje det er mest sannsynlig at det opprettes ny hendelse fordi et annet familiemedlem dør, og da er det jo uansett hensiktsmessig at det blir sendt brev, selv om det potensielt er koblet til "feil sak". 

Alternative løsninger blir å sjekke om det finnes behandling, i stedet for iverksatte som det gjøres i dag. 
Eller skrive om kontrollpunktene til å håndtere en liste med saker. 

Oddsen for at dette skjer er vel strengt tatt _veldig_ lav nå som det ikke lenger er mulig å opprette flere saker på samme person. Finnes potensielt et par tilfeller hvor noen har flere saker i produksjon allerede, ettersom det ikke var noen sperre før.

Reply:
Det er vel bare å sjekke i databasen det :D 
Har ikke hørt noe om det i hvert fall.

Det er sikkert ikke krise, men det kan jo potensielt bli at bruker tror at en søknad de har sendt inn ikke er sendt inn allikevel som gjør bruker usikker. Samtidig om det skjer sjeldent så er det vel noe vi kan ignorere...

Men burde nok konsolidere behandlingssjekken for saker og den for kontrollpunktene?

Kan jo splitte ut `hentSakMedBehandlinger` for en med flere saker som denne bruker og

Reply:
Litt usikker på om jeg forsto problemstillingen her. Selv om bruker har sendt inn en søknad, vil det vel fortsatt sendes et brev all tid denne søknaden ikke er ferdig behandlet? 

Denne sjekken som legges på vil vel kun gjøre at de som skulle ha to eller flere saker pga splitt ifm fnr-endring ikke feiler ved kjøring?